src/Core/Security/Authentication/Infrastructure/Api/Login/OpenId/LoginController.php

```diff
@@ -60,17 +62,26 @@ public function __invoke(
 
         $useCase($request, $presenter);
 
-        $response = $presenter->getPresentedData();
-        if ($response->getException() !== null) {
-            return View::createRedirect(
-                $this->getBaseUrl() . '/login?authenticationError=' . $response->getError()->getMessage()
-            );
+        switch (true) {
+            case is_a($presenter->getResponseStatus(), PasswordExpiredResponse::class)
```

Comment:
Password expiration is managed for OpenId ?

Reply:
No, but as the UseCase is Unique, the presenter in the useCase can present this response.